# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libptytty/hotfix.patch.cross
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libptytty-2.0/CMakeLists.txt.vanilla	2021-11-29 18:09:29.931154608 +0100
+++ libptytty-2.0/CMakeLists.txt	2021-11-29 18:11:00.746159293 +0100
@@ -271,28 +271,6 @@
   message(FATAL_ERROR "libptytty requires unix-compliant filehandle passing ability")
 endif()
 
-check_c_source_runs(
-  "
-   #include <grp.h>
-   #include <sys/stat.h>
-   #include <sys/types.h>
-   #include <unistd.h>
-
-   int main ()
-   {
-     struct stat st;
-     struct group *gr = getgrnam (\"tty\");
-     char *tty = ttyname (0);
-     return
-       !(gr
-         && tty
-         && stat (tty, &st) == 0
-         && st.st_gid == gr->gr_gid);
-   }
-  "
-
-  TTY_GID_SUPPORT)
-
 configure_file(
   config.h.cmake
   config.h)
